python_modules/libraries/dagster-databricks/dagster_databricks/configs.py

```diff
@@ -607,6 +605,14 @@ def define_databricks_storage_config():
     )
 
 
+def define_databricks_env_variables():
+    return Field(
+        Permissive(),
+        description="Dictionary of arbitrary environment variables",
```

Comment:
This description could be a bit more specific, maybe "Dictionary of arbitrary environment variables to be set on the databricks cluster"?